assets/javascripts/wizard/models/custom.js.es6

```diff
@@ -110,12 +94,45 @@ export function findCustomWizard(wizardId, params = {}) {
       Discourse.Site.currentProp("categoriesById", categoriesById);
       Discourse.Site.currentProp(
         "uncategorized_category_id",
-        wizard.uncategorized_category_id
+        wizardJson.uncategorized_category_id
       );
     }
 
-    return CustomWizard.create(wizard);
+    return CustomWizard.create(wizardJson);
+  },
+});
+
+export function findCustomWizard(wizardId, params = {}) {
+  let url = `/w/${wizardId}`;
+
+  let paramKeys = Object.keys(params).filter((k) => {
+    if (k === "wizard_id") return false;
+    return !!params[k];
+  });
+
+  if (paramKeys.length) {
+    url += "?";
+    paramKeys.forEach((k, i) => {
+      if (i > 0) {
+        url += "&";
+      }
+      url += `${k}=${params[k]}`;
+    });
+  }
+
+  return ajax({ url, cache: false, dataType: "json" }).then((result) => {
+    return CustomWizard.build(result);
   });
 }
 
+var _wizard_store;
+
+export function updateWizard(wizard) {
```

Comment:
Similarly, the naming convention should suggest we're updating the local copy.

Reply:
@fzngagan Agreed on this one :+1: Updated.